### PR TITLE
Improve the mission statement

### PR DESCRIPTION
**Pull request by clairernovotny**
A mission statement should be short and sweet and describe the primary goal or reason for the Foundation's existence. We can elaborate in a vision statement.

### CODE REVIEW

File: input/index.cshtml [hunk 1]
```diff
@@ -10,7 +10,7 @@ isHome: true
         <div id="home-hero_content" class="position-relative">
             <h2 id="home-hero_title">Independent. Innovative. Always open source.</h2>
             <p id="home-hero_copy" class="mx-auto">
-                The .NET Foundation is an independent, non-profit organization established to support an innovative, commercially friendly, open-source ecosystem around the .NET platform.
+                The .NET Foundation is an independent, non-profit organization whose mission is to support an innovative, diverse, commercial-friendly, international open-source ecosystem for the .NET platform.
```
[the remark this answers is not in the document]
Comment (clairernovotny) [reply input/index.cshtml:13]: In this case, the intent is that business are able to use .NET-based OSS libraries without friction.

Clean IP and licensing is a key part of that, which is why the Foundation has projects sign a contribution agreement a CLA bot for future contributions. That ensures that no one is going to come out of the woodwork with a copyright claim on the code. It also means use of permissive licenses, which is one reason the Foundation does not support libraries with copyleft licenses.

It currently does not say anything about a project's commercial viability nor for sponsors of the Foundation, of which Microsoft is just one.

[the remark this answers is not in the document]
Comment (benaadams) [reply input/index.cshtml:13]: Is both.

If a project is not sustainable; then it's not "commercial-friendly" and the .NET Foundation should help enable business to give back to projects they use in a "commercial-friendly" way (as business purchasing can be a complicated internal system and a common barrier for all projects that the .NET Foundation should endeavor to ease).

Also the .NET Foundation does not support non-permissive licenses for it's libraries (like GPL) as they are hard to build on or use in a "commercial-friendly" way. (This is is different from GPL applications; which are used in a different way as they don't become part of the application like libraries do)

[the remark this answers is not in the document]
Comment (gortok) [reply input/index.cshtml:13]: @benaadams Non permissive licenses like the AGPL? I'll look through the bylaws and stuff; but are the licenses the .NET Foundation "is ok with" listed anywhere?

[the remark this answers is not in the document]
Comment (clairernovotny) [reply input/index.cshtml:13]: The bylaws do not specify every detail, that is not their purpose. The bylaws delegate to the projects policy, which is here https://github.com/dotnet-foundation/projects/blob/master/readme.md

[the remark this answers is not in the document]
Comment (gortok) [reply input/index.cshtml:13]: @clairernovotny  I'm wondering where those approved licenses are listed.  The page only says this:

" The code is offered under an open source license which has been approved by the .NET Foundation."

So what licenses have been approved?

[the remark this answers is not in the document]
Comment (clairernovotny) [reply input/index.cshtml:13]: That's at the discretion of the projects committee. In general, it's MIT and Apache 2.0. There's an open work item that the project committee is working on to further document those. That's in the committees notes here: https://github.com/orgs/dotnet-foundation/teams/projects/discussions/37.

Comment (BillWagner) [inline input/index.cshtml:13]: I'd re-order the last clause as follows:

```suggestion
                The .NET Foundation is an independent, non-profit organization whose mission is to support an international, diverse, innovative,  and commercial-friendly open-source ecosystem for the .NET platform.
```

**Comment by clairernovotny** [comment]
Closing as we'll resubmit this soon

**Pull request closed without merging**